packages/config/src/ProfileInfo.ts

```diff
@@ -359,6 +363,41 @@ export class ProfileInfo {
         this.ensureReadFromDisk();
         return this.mLoadedConfig;
     }
+
+    // _______________________________________________________________________
+    /**
+     * Create a zosmf session from profile arguments retrieved from
+     * ProfileInfo functions.
+     *
+     * @param profArgs
+     *      An array of profile arguments.
+     *
+     * @param connOpts
+     *      Options that alter our actions. See IOptionsForAddConnProps.
+     *      The connOpts parameter need not be supplied.
+     *      Default properties may added to any supplied connOpts.
+     *      The only option values used by this function are:
+     *          connOpts.requestToken
+     *          connOpts.defaultTokenType
+     *
+     * @returns A session that can be used to connect to zosmf;
+     */
+    public static createZosmfSession(
```

Comment:
It may be confusing to call this method `createZosmfSession`, since it doesn't use the ZosmfSession type (defined in https://github.com/zowe/zowe-cli/blob/master/packages/zosmf/src/ZosmfSession.ts), and nothing about this method seems specific to z/OSMF.

Reply:
good point - I guess similarly there is `initZosmfSessCfg`.  It seems like z/OSMF related topics would belong in Zowe CLI instead of our generic imperative infrastructure piece.